src/System.Net.Security/src/System/Net/NegotiateStreamPal.Unix.cs

```diff
@@ -53,12 +61,36 @@ internal static string QueryContextClientSpecifiedSpn(SafeDeleteContext security
 
         internal static SafeFreeCredentials AcquireDefaultCredential(string package, bool isServer)
         {
-            throw new PlatformNotSupportedException();
+            return AcquireCredentialsHandle(package, isServer, new NetworkCredential(string.Empty, string.Empty, string.Empty));
         }
 
         internal static SafeFreeCredentials AcquireCredentialsHandle(string package, bool isServer, NetworkCredential credential)
         {
-            throw new PlatformNotSupportedException();
+            if (isServer)
+            {
+                throw new NotImplementedException("AcquireCredentialsHandle: Server is not yet supported");
```

Comment:
Do we plan to implement it?  If not, this should be a PlatformNotSupportedException (and any string message should come from Strings.resx).